mosfit/models/magnetar/magnetar.json

```diff
@@ -149,37 +117,34 @@
             "vejecta"
         ]
     },
-    "envelopecore":{
+    "slsn":{
         "kind":"sed",
         "inputs":[
             "texplosion",
-            "redshift",
             "diffusion",
-            "temperature"
+            "vejecta",
+            "mejecta",
+            "kappa"
         ]
     },
     "extinction":{
         "kind":"sed",
         "inputs":[
-            "envelopecore",
-            "nhhost",
-            "ebv"
+            "slsn",
+            "nhhost"
         ],
         "requests":[
             ["bandwavelengths", "filters"],
-            [],
             []
         ]
     },
     "filters":{
         "kind":"observable",
         "inputs":[
-            "extinction",
```

Comment:
This is related to above, the `Filter` class needs to know that `lumdist` is a parameter that can be passed, so these lines should remain unless it was breaking things (in which case we should fix the breakage!)